lib/spack/docs/config_yaml.rst

```diff
@@ -145,6 +145,16 @@ hosts when making ``ssl`` connections.  Set to ``false`` to disable, and
 tools like ``curl`` will use their ``--insecure`` options.  Disabling
 this can expose you to attacks.  Use at your own risk.
 
+--------------------
+``custom_certs``
+--------------------
+
+Path to custom certificats for SSL verification. The value can be a 
+filesytem path, or an environment variable that expands to a file path.
+The default value is set to the environment variable ``SSL_CERT_FILE``
+to use the same syntax used by many other applications that automatically
+detect custom certificates.
```

Comment:
Unless we plan to add the configuration for the curl backend in this PR, I might suggest we add a caveat to point out this only works for urllib at the moment.

Reply:
We should make sure that this is translated over, if set, when spack runs curl:

https://stackoverflow.com/questions/38571099/how-can-i-set-the-certificates-in-curl

it would also be nice to respect SSL_CERT_DIR

Reply:
Updated with a path for curl.